tests/integration/ovs_test.py

```diff
@@ -103,3 +103,17 @@ def test_create_and_remove_ovs_bridge_with_internal_port_and_static_ip():
 
     assertlib.assert_absent(BRIDGE1)
     assertlib.assert_absent(PORT1)
+
+
+@pytest.mark.xfail(
+    raises=(NmstateLibnmError, AssertionError),
+    reason='https://bugzilla.redhat.com/1724901',
```

Comment:
This reason does not match this particular case. The problem here is that the interface name is a primary key for Nmstate/needs to be unique. To handle this, Nmstate needs to use a tuple like (name, type) as the primary key or do some special handling for OVS.